src/Invoice.php

```diff
@@ -58,6 +60,46 @@ public function get($withItems, $headers = null)
         return $object;
     }
 
+    /**
+     * Get payments for this invoice
+     *
+     * @param  boolean       $withPluginInfo
+     * @param  boolean       $withAttempts
+     * @param  string[]|null $headers        Any additional headers
+     * @return InvoicePayment[]
```

Comment:
There should be a spacing line between "@param"s and the "@return".

Also, the "Any additional headers" should be one more space away to the right, because it has to be column aligned with the end of "Info" above.

Reply:
ok. I followed the space requirements given by phpcs.

Reply:
@GregOriol 

> Also, the "Any additional headers" should be one more space away to the right, because it has to be column aligned with the end of "Info" above.

Sorry, PHPCS complaints about adding an extra space before "Any additionnal headers" : https://travis-ci.org/killbill/killbill-client-php/jobs/306525628

Reply:
PHPCS is probably right then :-)